packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/CommitForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ChangeEvent } from 'react';
+import { Text, Stack, Input, Textarea, Button } from '@codesandbox/components';
+import { useOvermind } from 'app/overmind';
+
+export const CommitForm = () => {
+  const {
+    actions: {
+      git: {
+        createCommitClicked,
+        createPrClicked,
+        descriptionChanged,
+
+        subjectChanged,
+      },
+    },
+    state: {
+      editor: { isAllModulesSynced },
+      git: { description, originalGitChanges: gitChanges, subject },
+    },
+  } = useOvermind();
+  const hasWriteAccess = (rights: string) =>
+    ['admin', 'write'].includes(rights);
+
+  const modulesNotSaved = !isAllModulesSynced;
+
+  const changeSubject = ({
+    target: { value },
+  }: ChangeEvent<HTMLInputElement>) => subjectChanged({ subject: value });
+
+  const changeDescription = ({
+    target: { value },
+  }: ChangeEvent<HTMLTextAreaElement>) =>
+    descriptionChanged({ description: value });
+
+  return (
+    <>
+      <Text size={3} block marginBottom={2} marginX={2}>
```

Comment:
This could be replaced by FormField. Example: https://pr3402.build.csb.dev/s/wandering-cache-s791s-dedqt